Monacs.Core/Unit/Result.Unit.Extensions.cs

```diff
@@ -4,14 +4,27 @@
 
 namespace Monacs.Core.Unit
 {
+    ///<summary>
+    /// Extensions for Result<Unit> type.
+    ///</summary>
     public static class Result
     {
+        ///<summary>
+        /// Creates successful Result<Unit>.
+        ///</summary>
         public static Result<Unit> Ok() =>
             Core.Result.Ok(Unit.Default);
```

Comment:
How do you find using `using static <class>` ? Not sure if it'd help here or just complicate things, so just asking.

Reply:
I think it's better to leave it in quick start and not pollute API docs with usage details.